README.md

```diff
@@ -24,10 +24,9 @@ Last tested on OS API `2.1.8` and Home Assistant `0.95.0`
       host: <host>
       password: <md5-password>
     ```
-    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.
-    - Replace `<md5-password>` with the MD5 encrypted version of your password to your Opensprinkler interface.
-
-3. Restart Home Assistant
+    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.  This should include the port number as well, such as `8080`, the default that OpenSprinkler uses.
```

Comment:
could you quickly add an example 
```
such as `<host>:8080` for port `8080`, the default...
```

Reply:
I added a little more detail than that given #44 that you just linked to.